modules/gdpr_fields/src/EntityTraversalInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Drupal\gdpr_fields;
+
+/**
+ * Defines a common interface for dependency container injection.
```

Comment:
Looks like this comment is copy/paste from `EntityTraversalContainerInjectionInterface`. Can it be updated to be specific to this task?